frontend/src/components/Footer.tsx

```diff
@@ -0,0 +1,17 @@
+const Footer = () => {
+  return (
+    <div className=" bg-orange-500 py-10">
+      <div className="container mx-auto flex flex-col md:flex-row justify-between items-center">
+        <span className="text-3xl text-white font-bold tracking-tight">
+          Eats.com
+        </span>
+        <span className="text-white font-bold tracking-tight flex gap-4">
+          <span>Privacy Policy</span>
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/12#issuecomment-2359035189>#36a063</a><span> - 09/18/2024, 05:36 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The "Privacy Policy" and "Terms of Service" spans are not implemented as interactive elements (e.g., links or buttons), which reduces accessibility and usability.
</div>
<div id="fix">
<b>Fix:</b> Implement the "Privacy Policy" and "Terms of Service" text as anchor (&lt;a&gt;) elements with appropriate href attributes. Also, consider extracting these as separate components or configuration for better maintainability.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -6,10 +6,10 @@
        Eats.com
      </span>
      <span className="text-white font-bold tracking-tight flex gap-4">
 -      <span>Privacy Policy</span>
 -      <span>Terms of Service</span>
 +      <a href="/privacy-policy" className="hover:underline">Privacy Policy</a>
 +      <a href="/terms-of-service" className="hover:underline">Terms of Service</a>
      </span>
    </div>
  </div>
 );
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged